packages/deployments/contracts/contracts/core/connext/helpers/RelayerProxyHub.sol

```diff
@@ -390,6 +384,8 @@ contract RelayerProxyHub is RelayerProxy {
 
     if (fromChain == 137 || fromChain == 80001) {
       IPolygonHubConnector(hubConnectors[fromChain]).receiveMessage(encodedData);
+    } else {
+      revert RelayerProxyHub__processFromRoot_unsupportedChain(fromChain);
```

Comment:
wont this always fall through and revert? i.e. if sending from optimism, it wouldn't exit early and fall into this edgecase. we likely need to return early out of the other functions

Reply:
OMG you're right. Need elses.